articles/tutorials/google-cloud-platform.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Securing Google Cloud Endpoints with Auth0
+description: How to secure a Google Cloud Endpoints API with Auth0.
+toc: true
+---
+
+# Securing Google Cloud Endpoints with Auth0
+
+[Google Cloud Endpoints (GCE)](https://cloud.google.com/endpoints/) is an API management system providing features to help you create, maintain, and secure your APIs. GCE uses [OpenAPI](https://www.openapis.org/) to define your API's endpoints, input and output, errors, and security description.
+
+::: note
+For more information on the OpenAPI spec, see the [OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification) repository on GitHub.
+:::
+
+This tutorial will cover how to secure Google Cloud Endpoints with Auth0.
+
+## Prerequisites
+
+Before you begin you'll need a deployed GCE API. If you haven't already created an API, complete the [Cloud Endpoints Quickstart](https://cloud.google.com/endpoints/docs/quickstart-endpoints).
+
+The quickstart will walk you through creating a simple GCE API with a single endpoint, `/airportName`, that returns the name of an airport from its three-letter [IATA code](https://en.wikipedia.org/wiki/IATA_airport_code).
+
+```har
+{
+    "method": "GET",
+    "url": "https://YOUR_GCE_PROJECT.appspot.com/airportName",
+    "headers": [],
+    "queryString" : [
+      {
+        "name": "iataCode",
+        "value": "SFO"
+      }
+    ]
+}
+```
+
+## Define the API in Auth0
+
+Open [APIs > Auth0 Dashboard](${manage_url}/#/apis) and create a new API.
+
+![Create API](/media/articles/tutorials/gce-create-api.png)
+
+Make note of the **API Audience** identifier (`http://google_api` in the screenshot above) to use in the following step.
+
+## Update the API Configuration
+
+Next, we'll update the OpenAPI configuration file for the GCE API. For the sample API created during the quickstart this file is `openapi.yaml`.
+
+### Add Security Definitions
+
+Open the configuration file and add a new `securityDefinitions` section. In this section, add a new definition (`auth0_jwt`) with the following fields:
+
+Field | Description
+------|------------
+`authorizationUrl` | The authorization URL, should be set to `"https://${account.tenant}.auth0.com/authorize"`
```

Comment:
I think that instead of `"https://${account.tenant}.auth0.com/authorize"` you should use `"https://${account.namespace}/authorize"`

The difference is that if I have an EU account and my tenant's name is `mariap` then my domain will be `mariap.eu.auth0.com`. In this case, the vars will have the following values:
- `${account.tenant}` = `mariap`
- `${account.namespace}` = `mariap.eu.auth0.com`

(same comment applies to the other fields of this matrix as well)